packages/salesforcedx-vscode-core/package.json

```diff
@@ -211,6 +211,10 @@
           "command": "sfdx.force.apex.class.create",
           "when": "explorerResourceIsFolder && resourceFilename == classes && sfdx:project_opened"
         },
+        {
+          "command": "sfdx.force.folder.diff",
+          "when": "explorerResourceIsFolder && resourceFilename =~ /applications|aura|classes|components|contentassets|flexipages|layouts|lwc|objects|pages|permissionsets|staticresources|tabs|triggers$/ && sfdx:project_opened"
```

Comment:
@jag-sfdc If you remove the **resourceFilename** constraint it looks like the command will work for higher level folders. I'm not sure how to limit it to just project source folders, though. It does fail with an appropriate error message in that case.

Reply:
@rcoringrato-sfdc - I have no idea why it hung indefinitely for me when I tried after the demo yesterday. It seems to be consistently working now.

Reply:
With Xiaoyi's merged changes, we're displaying the `SFDX: Diff File Against Org` command on folders. Once your PR is merged with his changes, are we going to be displaying both the `SFDX: Diff File Against Org` and `SFDX: Diff Folder Against Org` commands on folders? If so, we should remove the file diff command in folder contexts

Reply:
@jag-sfdc any thoughts about both the commands being present at the folder level?